### DISCUSSION TITLE
Why do we need this data?

### DISCUSSION THREAD

**Discussion by ActiveConclusion** [ActiveConclusion/COVID19_mobility#34] [Show and tell]
Well-collected data and beautiful visualizations - it's definitely very nice. But analytics for just analytics is usually useless in practice. So, my main question is next: how is possible to make a real call to action for communities/governments based on this data?
Feel free to share here your thoughts, your insightful dashboards and visualisations or your current researches.
Thanks!

**Comment by ladew222** [comment]
I wish I had an answer to that--I think all of us working here do.   With the help of John Hendy, my dashboard is now has the Google data and is [balefire.info](http://www.balefire.info). I am personally going to share al talk about it with peers and promote the use of the data as important to guiding policy and decisions.  But others may have more insights.

**Comment by ActiveConclusion** [comment]
@ladew222 Thanks for creating and sharing this delightful dashboard, excellent job! My small recommendation for UI is to change the date range slider to something like the [date range picker](https://www.daterangepicker.com/), everything else is pretty ok. Do you have plans to create a dashboard like this for other countries?

**Comment by ladew222** [comment]
Thank you. I will look into a date range picker. The current one which uses JQueryUI does not work with mobile so another reason to switch.  It would be good to have one for other countries or a global version if I could pull it off. I know the google data would be there. I do not know how to pull demographic info but I will investigate.  My background is in US geospatial analysis, so I have relied heavily on NHGIS out of Minnesota. I might need to talk to demographers from some other countries to see what is best for that information. Great idea. Thinking. As an FYI,  i shared the project with  the university where I work  and some people are talking to people they know in the media, so here is to getting the word out. I will continue to promote

**Comment by jwhendy** [comment]
My brother and I started putzing on data. He initially inquired about the relationship of population density to the spread of COVID-19. I figured that was a rather trivial case to make, as population density should exponentially increase interactions and shared spaces... but then it dawned on me that stay-at-home orders should serve as an effective reduction in density.

Our idea morphed into what amounts to an outlier detection system, but that brought about the realization that to identify an outlier, you need a reasonable model that tells you what some value _should_ be. This brought about starting to pull census data. I was curious what effect population density, age, industry (farm vs. manufacturing vs tech), income, etc. might have.

With the US pretty varied in stay at home responses, I wondered if I could build an explorer of sorts to select counties with similar factors (like those above) and pull out those that are unexpectedly high. I even wondered about automating an email system that would then email local government... basically, across counties that seem to align on whatever factors we think are relevant, but let them know that theirs is x% higher than all the other ones. Maybe it's not apparent across the states who's actually winning this battle vs. who's losing given the rapid pace/evolution? Thus, an identifier could maybe get the word to folks before they have to start seeing the death counts rack up.

The most basic hypothesis being put forward thus far is: social distancing is key to slowing growth.

I think this mobility data is one of the most accurate signs of that, so that's where I hoped to start. Can we find a trend between activity and spread. 

This plot is from maybe a week ago based on the NY Times US county level case data. I filtered each county to more than 10 cases, then solved for the exponent coefficient, `b`: `cases = a*10^(b*days)`. This probably is a kludge, but I wanted to get at the rate of increase and `b` seemed like a way to unify this comparison. That's plotted against `rho`, population/square_miles. We thought it was interesting that the highest `b` values were orders of mag lower in population density...

![exp-vs-density](https://user-images.githubusercontent.com/379357/79053377-8d435380-7c02-11ea-9b49-8f51ea0289cd.png)

This isn't the easiest to see, but I tried visualizing cases vs. time, colored by density. The highest ~~slopes~~ [edit: case densities] are not yellowish...

![date-vs-cases](https://user-images.githubusercontent.com/379357/79053412-c4196980-7c02-11ea-83de-97b7db12dccd.png)

I plan to next try and merge these with this mobility data. Sadly, I'm fearing that if there are not obvious trends... the case/death data is just so awful and inconsistent, it might be a lost cause to try and make predictions.

**Comment by jwhendy** [comment]
This is the time series data for CA and NJ (earliest stay at home orders, 03-19 and 03-21)  vs. FL and GA (some of the latest orders, 04-03). How does this make any sense?

![mobility-by-segment_ca-nj-vs-fl-ga](https://user-images.githubusercontent.com/379357/79055074-a1418200-7c0f-11ea-9fa0-df307558bfad.png)

In particular, look at workplace vs. residential. _Everyone_ starts their coordinated decrease/increase ~2020-03-15... shouldn't this not be so consistent across these states? Or does this mean that businesses were already acting on their own before statewide orders went into effect? I'm perplexed.

**Edit:** also, very interesting to see split bands in FL and GA residential... I wonder if those are healthcare workers who had to start responding. Some group of people specifically was _decreasing_ their time at home while most were _increasing_. What's more odd is in the original time series data for all counties (through 2020-03-29), I saw the same split bands in workplace, but now I don't.

**Comment by jwhendy** [comment]
I just read [an article](https://www.vox.com/2020/4/7/21205890/coronavirus-covid-19-pandemic-new-york-california) which prompted me to post the above visualization (after replacing NJ with NY) on [reddit](https://www.reddit.com/r/dataisbeautiful/comments/g1g6q8/oc_ca_ny_fl_and_ga_activity_levels_compared_using/). There may be some relevant discussion that emerges.

**Comment by ladew222** [comment]
Interesting.  I am working on adding more filters to graphs and hopefully the maps.  Remaking a bit of the data-handling to handling to make more feature requests possible.  I am planning to create the ability to chart and compare state totals.  I may also add the ability to compare counties, but will let requests guide me. If there are any feature requests, let me know. I also want to be careful not break what I have as we are gaining users each day, so I may be a bit slower to roll out. I will probably send some more information out now that are data and usability is in a good place. Thanks

**Comment by jwhendy** [comment]
I saw @ActiveConclusion mention the Apple data. Has that been compared to this? The core point from above which hasn't been addressed: **What does it mean that states with severely delayed stay at home orders have identical mobility trends to the earliest states with these orders?**

Either way I see a dead end:

- the data is accurate: the same mobility changes across all states mean it is an irrelevant factor. This would be akin to knowing that age is a relevant factor, pulling census data, and finding that every county in the US has an identical age distribution. It's data, it's just not helpful or relevant to any insights.
- the data if inaccurate: we're better off modeling by stay at home date and ignoring this altogether.

Here's the per-state view. I'm at a loss. How is there not one state that shows staying steady with the baseline as of ~Mar 15? FL and GA didn't get orders until ~6 weeks after this on Apr 03! I believe this is the key objectio nto "why do we need this data." It either doesn't allow differentiating states by mobility changes, or it's plain wrong.

![state-mobility](https://user-images.githubusercontent.com/379357/79499861-30c0a980-7ff1-11ea-9736-886eadfd2d61.png)

**Comment by cipolataman** [comment]
How we use the data - see [https://www.agility.asia/covid] for analysis, discussion and projections across 26 countries: 
1) Cross-correlating mobilities against one another to determine their correlative link pre and post lockdowns - useful for generating hypothesis on how mobilities related to one another, e.g. use of transit to get to work, shops etc 
2) Using above to determine differences across countries/regions, particularly for guiding which mobilities matter more / less to control well/stringently in a lockdown situation
3) Cross-correlating with death and infection statistics to determine median time for case detection and median time of infection to death, amongst other things. Rolling correlation can determine how case detection may have improved/deteriorated with time 
4) Timing the exact date of onset of lockdowns, and using the mobility level (via correlation) to help model the transmission rate decline 
5) Improving predictability of future forecasts, by using mobility changes to determine potential transmission rate changes through a correlative factor - this can be computed, and assumed to adjust downwards with time as social distancing, personal hygiene etc is improved through learning / habit forming 
6) Keeping us busy and helping others who may find the analysis useful! 

Thank you a great deal for compiling it. It has been tremendously helpful. I just wish Google's data was updated more frequently - it constantly lags significantly behind reporting day.

**Comment by ActiveConclusion** [comment]
@cipolataman Great research, well done! Thank you for sharing your results, I've also subscribed for new updates :) I'll try to explore deeper the research in free time.

**Comment by Damjanv1** [comment]
Hey there I use the Apple data for a dashboard that I have created for my home state in Australia

https://public.tableau.com/profile/damjan.vlastelica#!/vizhome/CovidNSWTracker/HomeDash?publish=yes

**Comment by ActiveConclusion** [comment]
Hey @Damjanv1! Nice dashboard, very detailed and insightful. Thanks for sharing!

**Comment by jiweiqi** [comment]
Thanks a lot. We have used Google/Apple mobility data to predict future US Fuel demand. The results can be found at https://covid19-mobility.com.

**Comment by Damjanv1** [comment]
> Thanks a lot. We have used Google/Apple mobility data to predict future US Fuel demand. The results can be found at https://covid19-mobility.com.

That's smart.. Looks good thanks for sharing

**Comment by ActiveConclusion** [comment]
@jiweiqi Wow, great work! This is a non-trivial way to use this data, but it looks simple and logical. I would be interested to look at some quality metrics of the model. Thanks for sharing!

**Comment by jiweiqi** [comment]
@ActiveConclusion Yeah. That'a really a good suggestion. I am thinking of presenting the mobility index projection state by state, and we can show the R-square state by state.
